src/app/routes/home/getInitialData/index.js

```diff
@@ -1,11 +1,11 @@
 import pipe from 'ramda/src/pipe';
 import pathOr from 'ramda/src/pathOr';
 import fetchPageData from '../../utils/fetchPageData';
-import filterUnknownContentTypes from './filterUnknownContentTypes';
-import filterEmptyGroupItems from './filterEmptyGroupItems';
-import squashTopStories from './squashTopStories';
-import addIdsToItems from './addIdsToItems';
-import filterGroupsWithoutStraplines from './filterGroupsWithoutStraplines';
+import filterUnknownContentTypes from '../../utils/filterUnknownContentTypes';
```

Comment:
It might be better to do:

```suggestion
import filterUnknownContentTypes from '#app/routes/utils/filterUnknownContentTypes';
```

Reply:
Same with the rest of imports

Reply:
Yes, this is done!